VMS/compilerconfig.json

```diff
@@ -94,5 +94,25 @@
   {
     "outputFile": "Pages/ActivityInfoPage/OtherActivities.razor.css",
     "inputFile": "Pages/ActivityInfoPage/OtherActivities.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Organization-Profile/ActTookPlace.razor.css",
+    "inputFile": "Pages/Organization-Profile/ActTookPlace.razor.scss"
+  },
+  {
+    "outputFile": "Pages/Organization-Profile/ActFavourite.razor.css",
+    "inputFile": "Pages/Organization-Profile/ActFavourite.razor.scss"
```

Comment:
Seems like you delete this component. If any component is deleted, please remove it here in this file as well.